README.rst

```diff
@@ -6,22 +6,41 @@ pseudopeople is a Python package that generates realistic simulated data about a
 fictional United States population, designed for use in testing entity resolution
 (record linkage) methods or other data science algorithms at scale.
 
-Supported Python versions: 3.10, 3.11
+.. _python_support:
 
-You can install pseudopeople from PyPI with pip:
+**Supported Python versions: 3.10, 3.11**
 
-  ``> pip install pseudopeople``
+.. _end_python_support:
 
-or build it from source with
+Installation
+============
+
+.. _installation:
+
+Once Python is installed, you can install pseudopeople with `pip <https://pip.pypa.io/en/stable/>`_ 
+by running the command:
+
+.. highlight:: console
+
+::
 
-  ``> git clone https://github.com/ihmeuw/pseudopeople.git``
+  $ pip install pseudopeople
 
-  ``> cd pseudopeople``
+Or, you can install from source on `the pseudopeople GitHub repository <https://github.com/ihmeuw/pseudopeople>`_.
 
-  ``> python setup.py install``
+.. highlight:: console
+
+::
+
+  $ git clone https://github.com/ihmeuw/pseudopeople.git
+  $ cd pseudopeople
+  $ python setup.py install
```

Comment:
Is this a recommended way to install these days? I'd probably say `pip install git+https://github.com/ihmeuw/pseudopeople.git`

Reply:
I thought that was a weird bit of instructions but didn't know enough to change it.